stdlib/3/unittest/mock.pyi

```diff
@@ -10,60 +10,60 @@ if sys.version_info >= (3, 3):
 
     class _SentinelObject:
         name = ...  # type: Any
-        def __init__(self, name): ...
+        def __init__(self, name: Any) -> None: ...
 
     class _Sentinel:
-        def __init__(self): ...
-        def __getattr__(self, name): ...
+        def __init__(self) -> None: ...
+        def __getattr__(self, name: str) -> Any: ...
 
     sentinel = ...  # type: Any
     DEFAULT = ...  # type: Any
 
     class _CallList(list):
-        def __contains__(self, value): ...
+        def __contains__(self, value: Any) -> bool: ...
 
     class _MockIter:
         obj = ...  # type: Any
-        def __init__(self, obj): ...
-        def __iter__(self): ...
-        def __next__(self): ...
+        def __init__(self, obj: Any) -> None: ...
+        def __iter__(self) -> Any: ...
+        def __next__(self) -> Any: ...
 
     class Base:
-        def __init__(self, *args, **kwargs): ...
+        def __init__(self, *args: Any, **kwargs: Any) -> None: ...
 
-    class NonCallableMock(Base):
-        def __new__(cls, *args, **kw): ...
-        def __init__(self, spec=None, wraps=None, name=None, spec_set=None, parent=None, _spec_state=None, _new_name='', _new_parent=None, _spec_as_instance=False, _eat_self=None, unsafe=False, **kwargs): ...
-        def attach_mock(self, mock, attribute): ...
-        def mock_add_spec(self, spec, spec_set=False): ...
+    class NonCallableMock(Any):
```

Comment:
Inheriting from Any is an interesting choice. I'm not sure what type checkers will make of it.